recipes/raylib/all/conanfile.py

```diff
@@ -72,14 +83,29 @@ def generate(self):
             tc.variables["USE_EXTERNAL_GLFW"] = "ON"
             tc.variables["OPENGL_VERSION"] = "OFF" if not self.options.opengl_version else self.options.opengl_version
         tc.variables["WITH_PIC"] = self.options.get_safe("fPIC", True)
+
+        tc.variables["CUSTOMIZE_BUILD"] = "ON"
+        on_off = lambda x: "ON" if x else "OFF"
+        tc.variables["SUPPORT_MODULE_RAUDIO"]    = on_off(self.options.module_raudio)
+
+        # this makes it include the headers rcamera.h, rgesture.h and rprand.h
+        tc.variables["SUPPORT_CAMERA_SYSTEM"]    = "ON"
+        tc.variables["SUPPORT_GESTURES_SYSTEM"]  = "ON"
+        tc.variables["SUPPORT_RPRAND_GENERATOR"] = "ON"
+
+        tc.variables["SUPPORT_EVENTS_WAITING"]       = on_off(self.options.events_waiting)
+        tc.variables["SUPPORT_CUSTOM_FRAME_CONTROL"] = on_off(self.options.custom_frame_control)
+
```

Comment:
```suggestion

        tc.variables["CUSTOMIZE_BUILD"] = self.options.customize_build
        tc.variables["SUPPORT_MODULE_RAUDIO"] = self.options.get_safe("module_raudio")
        tc.variables["SUPPORT_EVENTS_WAITING"] = self.options.get_safe("events_waiting")
        tc.variables["SUPPORT_CUSTOM_FRAME_CONTROL"] = self.options.get_safe("custom_frame_control")
```

I would ask to remove this forced customization and simplify it instead:

- The upstream goes to the opposite way by default: https://github.com/raysan5/raylib/blob/5.0/CMakeOptions.txt#L11. The `CUSTOMIZE_BUILD` disabled, same for camera, gestures and rprand
- Not even others package managers are customizing it, real example:
  - AUR: https://gitlab.archlinux.org/archlinux/packaging/packages/raylib/-/blob/main/PKGBUILD?ref_type=heads
  - Fedora: https://src.fedoraproject.org/rpms/raylib/blob/rawhide/f/raylib.spec
  - Homebrew: https://github.com/Homebrew/homebrew-core/blob/79b0412c280d8a1283a2f38f444cad436261e1c9/Formula/r/raylib.rb